cmd/lakectl/cmd/docs.go

```diff
@@ -84,6 +84,25 @@ the Docker container. That is to say, ` + "`localhost`" + ` to a Docker containe
 ## Command Reference
 `
 
+var cliReferenceHiddenCommandsSeparator = `
+-------
+
+## Undocumented commands
```

Comment:
A thought:
This section is pretty much at the top of the file, so it pretty much pops up. 
If it can be pushed to the bottom of the file, might be nicer I think…

Reply:
Not sure, could you explain on the auto-generated CLI reference file in this PR where things should go?

Reply:
Well, I can't think of another option actually...